### PR TITLE
Add timeToFirstContentfulPaint and timeToFirstMeaningfulPaint metrics

### PR DESCRIPTION
**Pull request by stephendonner**
These metrics are needed to (eventually) also calculate Time to Consistently Interactive: https://github.com/WPO-Foundation/webpagetest/blob/c53214c24a99f52add146eba6aec1cd137a4bcee/docs/Metrics/TimeToInteractive.md#time-to-consistently-interactive-calculation

@davehunt r?

### CODE REVIEW

File: metrics.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [
   {"name": "TTFB", "description": "Time to First Byte", "unit": "milliseconds"},
   {"name": "render", "description": "startRender", "unit": "seconds"},
-  {"name": "firstPaint", "description": "Time to First Non-Blank Paint", "unit": "seconds"},
+  {"name": "firstPaint", "description": "Time to First Non-Blank Paint", "unit": "milliseconds"},
```
Comment (davehunt) [inline metrics.json]: Just checking, is this change intentional?

File: metrics.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+  {"name": "timeToFirstMeaningfulPaint", "description": "Time to First Meaningful Paint", "unit", "milliseconds"},
```
Comment (davehunt) [inline metrics.json]: See the highlight, this should be a colon and not a comma.

File: metrics.json [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+  {"name": "timeToFirstContentfulPaint", "description": "Time to First Contentful Paint", "unit", "milliseconds"),
```
Comment (davehunt) [inline metrics.json]: Please see the hightlighted characters, which should be comma and closing curly brace.

**Review by davehunt — CHANGES_REQUESTED** [review]

**Comment by davehunt** [comment]
Was this meant to be opened against my fork/branch @stephendonner?

File: metrics.json (hunk 1 above)
Comment (stephendonner) [reply metrics.json]: Originally it was, but I've double-checked Chrome Lighthouse and other tooling, and they tend to list in seconds.

Thanks for noticing, and I'll change!

File: metrics.json (hunk 3 above)
Comment (stephendonner) [reply metrics.json]: Fixed.

**Comment by stephendonner** [comment]
> Was this meant to be opened against my fork/branch @stephendonner?

Originally, yes, as I was hoping we could do something like: davehunt/wpt-api job-dsl-dashboards -> mozilla/wpt-api job-dsl -> mozilla/wpt-api master.

I'll re-target after I hopefully sort out the branches, today/tomorrow!

File: metrics.json (hunk 2 above)
Comment (stephendonner) [reply metrics.json]: Fixed.

**Comment by stephendonner** [comment]
I added ```timeToContentfulPaint``` and ```timeToFirstInteractive``` over in https://github.com/mozilla/wpt-api/blob/10358e2fc3ca7cb667d0566ebf79f0fca4b9a988/metrics.json#L5 and https://github.com/mozilla/wpt-api/blob/10358e2fc3ca7cb667d0566ebf79f0fca4b9a988/metrics.json#L7, respectively.

This was done as part of https://github.com/mozilla/wpt-api/pull/180.
